### DISCUSSION TITLE
Xp bottle

### DISCUSSION THREAD

**Discussion by KeifCheif** [DiscordDungeons/Bugs#1015] [Ideas]
A bottle of experience obtained through brewing assorted resources together using #!brew which will give you or your pet an amount of xp that doesn't scale to your or your pet's level. The recipe would be uncommon-to-rare resources, and would be affected by gear enchanted with sapphires, or rings that boost XP earned.

**Comment by FelixArgyle** [comment]
not gonna be a thing since directly giving "xp" is just bad, boost in xp is just better _(might aswell just say, let us buy xp pretty much)_
